src/main/java/com/onaio/steps/model/ODKForm/ODKSavedForm.java

```diff
@@ -41,7 +41,7 @@
     String jrVersion;
 
 
-    protected ODKSavedForm(String id, String jrFormId, String displayName, String jrVersion, String instanceFilePath, String status) {
+    public ODKSavedForm(String id, String jrFormId, String displayName, String jrVersion, String instanceFilePath, String status) {
```

Comment:
Was there a need to change this scope?

Reply:
i thought did was some testing forgot to revert. Not reverted.